mort_cal.py

```diff
@@ -0,0 +1,38 @@
+#fixed variables
+## Fixed variables are mostly unliked, as we don't have good reasons typically to make these
+## fixed vars. Usually, you want this passed in by a user or you want the application to read
+## the variables from a config file, where they can be modified instead
+interest_rate = 3.75
```

Comment:
generally, you want to use Pascal case for variables. Snake case should be avoided as it is not a standard for vars:
[https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841](url)